libs/liquidity/src/lib/liquidity-manager.tsx

```diff
@@ -0,0 +1,132 @@
+import { useRef, useMemo } from 'react';
+import {
+  AsyncRenderer,
+  ResizableGrid,
+  ResizableGridPanel,
+  Tab,
+  Tabs,
+} from '@vegaprotocol/ui-toolkit';
+
+import type { AgGridReact } from 'ag-grid-react';
+import { LiquidityTable } from './liquidity-table';
+import type { MarketLiquidity } from './__generated__/MarketLiquidity';
+import { useQuery } from '@apollo/client';
+import classNames from 'classnames';
+import { t } from '@vegaprotocol/react-helpers';
+import { LiquidityProvisionStatus } from '@vegaprotocol/types';
+import {
+  MARKET_LIQUIDITY_QUERY,
+  useLiquidityProvision,
+} from './liquidity-data-provider';
+
+export const SISKA_NETWORK_PARAMETER = 'market.liquidity.stakeToCcySiskas';
+
+interface LiquidityManagerProps {
+  partyId?: string;
+  marketId: string;
+}
+
+export const LiquidityManager = ({
+  partyId,
+  marketId,
+}: LiquidityManagerProps) => {
+  const gridRef = useRef<AgGridReact | null>(null);
+  const {
+    data: marketLiquidityData,
+    loading,
+    error,
+  } = useQuery<MarketLiquidity>(MARKET_LIQUIDITY_QUERY, {
+    variables: useMemo(() => ({ marketId }), [marketId]),
+  });
+
+  const { liquidityProviders, suppliedStake, targetStake, code, symbol } =
+    useLiquidityProvision({ data: marketLiquidityData });
```

Comment:
suggestion (non blocking): IMO I'd keep the query and the data manipulation all in one hook. So components that need to use the data can consume it more simply

Reply:
nice! updated.